pkgs/development/python-modules/flask-expects-json/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flask
+, jsonschema
+}:
+
+buildPythonPackage rec {
+  pname = "flask-expects-json";
+  version = "1.7.0";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "Fischerfredl";
+    repo = pname;
+    rev = version;
+    hash = "sha256-CUxuwqjjAb9Fy6xWtX1WtSANYaYr5//vY8k89KghYoQ=";
+  };
+
+  propagatedBuildInputs = [ flask jsonschema ];
+
+  # tests aren't included in sdist and don't pass on current flask
+  doCheck = false;
```

Comment:
It's possible to execute some tests, see https://github.com/NixOS/nixpkgs/pull/218018/commits/65f3d3bd9c8f196afb0b4baeccbd499ebaba187d

Reply:
@SuperSandro2000 What abou this review comment?

Reply:
Pushed as a follow-up change in #220613.